hoot-core/src/main/cpp/hoot/core/visitors/AddMeasurementTagsVisitor.cpp

```diff
@@ -56,31 +55,31 @@ HOOT_FACTORY_REGISTER(ElementVisitor, AddMeasurementTagsVisitor)
 
 void AddMeasurementTagsVisitor::visit(const ElementPtr& pElement)
 {
-  if( !_addArea && !_addWidth && !_addArea ) return;
+  if (!_addArea && !_addWidth) return;
```

Comment:
This was a typo and should be:
if (!_addArea && !_addWidth && !_addLength) return;